conf/app.routes

```diff
@@ -1,3 +1,2 @@
 # microservice specific routes
-
-GET        /hello-world             uk.gov.hmrc.oasdiscoveryproxy.controllers.MicroserviceHelloWorldController.hello
+-> / uk.gov.hmrc.oasdiscoveryproxy.controllers.OasDiscoveryRouter
```

Comment:
should this not be `/internal`, based on the [OAS file](https://github.com/hmrc/integration-catalogue-oas-files/blob/main/platforms/hip/publish-files/oas-discovery-api.yaml#L19)?